lib/absinthe/phase/document/arguments/data.ex

```diff
@@ -49,8 +49,8 @@ defmodule Absinthe.Phase.Document.Arguments.Data do
   def handle_node(%Input.Value{normalized: %Input.Object{fields: fields}} = node) do
     data =
       fields
-      |> Enum.filter_map(
-        &include_field?/1,
+      |> Enum.filter(&include_field?/1)
+      |> Enum.map(
         &{&1.schema_node.__reference__.identifier, &1.input_value.data}
```

Comment:
@benwilson512 Whoa… that's really nice! I'm not super familiar with comprehensions yet, so I tend not to reach for them, but this is 👌

I've force pushed the branch with the changes you've asked for, which, it seems, is the wrong way to handle this requested changes stuff on Github… 👻